shell/src/test/java/alluxio/cli/bundler/InfoCollectorTestUtils.java

```diff
@@ -37,12 +44,37 @@ public static File createTemporaryDirectory() {
   }
 
   public static File createFileInDir(File dir, String fileName) throws IOException {
-    File newFile = new File(Paths.get(dir.getAbsolutePath(), fileName).toString());
+    File newFile = new File(Paths.get(dir.getCanonicalPath(), fileName).toUri());
     newFile.createNewFile();
     return newFile;
   }
 
+  public static File createDirInDir(File dir, String dirName) throws IOException {
+    File newDir = new File(Paths.get(dir.getCanonicalPath(), dirName).toUri());
+    newDir.mkdir();
+    return newDir;
+  }
+
   public static void create() {
     Files.createTempDir();
   }
+
+  public static void verifyAllFiles(File targetDir, Set<String> expectedFiles) throws IOException {
+    Set<String> copiedFiles = getAllFileNamesRelative(targetDir, targetDir);
+    assertEquals(expectedFiles, copiedFiles);
```

Comment:
This invokes `Set#equals` which compares the size and checks existence of all elements.